gradle.properties

```diff
@@ -12,3 +12,5 @@ kotlin.incremental=true
 # Kotlin Test configuration
 #Parallelism needs to be set to 1 since the concurrent tests in arrow-effects become flaky otherwise
 kotlintest.parallelism=1
+KOTLIN_VERSION=1.3.70
+KOTLIN_TEST_VERSION=3.4.2
```

Comment:
Please, don't add those versions here. They come from `arrow` repository.

Reply:
Agree!! I'm closing this PR.. since bumping the versions worked and all test passed.. we can move on and update it in the general repo 👍